diff --git a/Remoting/ServerManager/CMakeLists.txt b/Remoting/ServerManager/CMakeLists.txt
index d9d4143f5a..7a348b0b53 100644
--- a/Remoting/ServerManager/CMakeLists.txt
+++ b/Remoting/ServerManager/CMakeLists.txt
@@ -202,6 +202,17 @@ vtk_module_add_module(ParaView::RemotingServerManager
   PRIVATE_HEADERS ${private_headers}
   TEMPLATE_CLASSES ${template_classes})
 
+if (TARGET absl::absl_log)
+  message(STATUS "ABSEIL_DLL absl::absl_log")
+  target_link_libraries (RemotingServerManager PRIVATE absl::absl_log absl::log_initialize absl::log_severity)
+else ()
+  find_library(ABSEIL_DLL NAMES abseil_dll)
+  message(STATUS "ABSEIL_DLL=${ABSEIL_DLL}")
+  if (ABSEIL_DLL)
+    target_link_libraries (RemotingServerManager PRIVATE ${ABSEIL_DLL})
+  endif ()
+endif ()
+
 paraview_protobuf_generate(
   PROTOS    vtkPVMessage.proto
   OUT_VAR   vtkPVMessage_sources
