diff --git a/CMake/ParaViewCompilerExtraFlags.cmake b/CMake/ParaViewCompilerExtraFlags.cmake
index 4feb06da0c..2e1e95c80f 100644
--- a/CMake/ParaViewCompilerExtraFlags.cmake
+++ b/CMake/ParaViewCompilerExtraFlags.cmake
@@ -37,7 +37,7 @@ if(CMAKE_COMPILER_IS_GNUCXX)
       set_target_properties(paraviewbuild
         PROPERTIES
         INTERFACE_LINK_OPTIONS
-        "LINKER:SHELL:${PARAVIEW_EXTRA_SHARED_LINKER_FLAGS} -lc ${CMAKE_SHARED_LINKER_FLAGS}")
+        "LINKER:SHELL:${PARAVIEW_EXTRA_SHARED_LINKER_FLAGS};SHELL:-lc ${CMAKE_SHARED_LINKER_FLAGS}")
     endif()
   endif()
 
